pkgs/top-level/all-packages.nix

```diff
@@ -21005,6 +21005,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreServices;
   };
 
+  MarbleMarcher = callPackage ../games/MarbleMarcher {};
```

Comment:
```suggestion
  marble-marcher = callPackage ../games/marble-marcher {};
```

Reply:
I though about making it this way too but was hesitant since the name used for the repo and for the executable was CamelCased. And I also found some examples like `nixpkgs.SDL`, `nixpkgs.SPAdes`, `nixpkgs.SkypeExport`, `nixpkgs.Sylk` and many others so I decided to keep it this way.

Do you really think it would be really a good idea since the executable name is `MarbleMarcher`? I find this rather confusing when package name is kind of similar but still the executable is quite not the same. Or maybe you would like to see the executable be renamed too?

Reply:
Then maybe rename the package (`pname`) as well?

Reply:
> Then maybe rename the package (pname) as well?

yes please.

> Do you really think it would be really a good idea since the executable name is MarbleMarcher?

The main binary name does not need to match the package name. If someone would search for the package he/she would probably search for marble and this would make it easier. I also think at least sdl should be renamed to be lowercase which makes it easier to find.

Reply:
@SuperSandro2000 
1. Renamed the directory name from `MarbleMarcher` to `marble-marcher`
2. Renamed the `pname` from `MarbleMarcher` to `marble-marcher`
3. Renamed the top-level name from `MarbleMarcher` to `marble-marcher`